plugins/woocommerce/client/blocks/assets/js/editor-components/utils/index.js

```diff
@@ -196,9 +196,41 @@ export const getCategory = ( categoryId ) => {
 	} );
 };
 
+/**
+ * Get a promise that resolves to a list of variation objects from the Store API
+ * and the total number of variations.
+ *
+ * @param {number} product Product ID.
+ * @param {Object} args    Query args to pass in.
+ */
+export const getProductVariationsWithTotal = ( product, args = {} ) => {
+	return apiFetch( {
+		path: addQueryArgs( `wc/store/v1/products`, {
+			type: 'variation',
+			parent: product,
+			orderby: 'title',
+			per_page: 100,
```

Comment:
Since we're making it possible to load more, do you think we should use less than `100` here?

Reply:
Yeah, good call! Done in a8ed741fbce71596a57497cd1b041192897e71a6.